src/actors/symbolication.rs

```diff
@@ -1043,7 +1042,44 @@ struct MinidumpState {
     assertion: String,
 }
 
+impl From<&ProcessState<'_>> for MinidumpState {
```

Comment:
_nit_: This should probably be an explicit `new` function rather than an implicit converter; especially, since it operates on a reference.

Reply:
Could you elaborate a bit more?

Reply:
Constructor functions are usually named `new` or `build` rather than via conversion traits. We're computing a state from a process state, not "casting" it.